bin/deploy.sh

```diff
@@ -21,14 +21,15 @@
 ## }
 
 #
-# OS Test 
-# $1 = OS 
+# OS Test
+# $1 = OS
 # Usage: if is_os 'darwin'; then
 #
 git status
-git pull -ff
```

Comment:
@arekkas Removed `-ff` (since the corresponding parameter is `--ff` and the default behaviour anyway). Added submodule updating to deploy script